library/src/options.cxx

```diff
@@ -182,6 +182,7 @@ options::options()
   this->Internals->init("ui.metadata", false);
   this->Internals->init("ui.font-file", std::string());
   this->Internals->init("ui.loader-progress", false);
+  this->Internals->init("ui.animation-progress", true);
```

Comment:
@Meakk should we change that to false and add it in the default config instead ?

Reply:
Yes I agree.

Reply:
So the option has to be set in `src/resources/configs/config.d/10_global.json`?

Reply:
Indeed! but you will need to update a few other tests that relies on the previous behavior.